holidays/countries/japan.py

```diff
@@ -60,7 +62,17 @@ def _populate(self, year):
             self[date(year, FEB, 23)] = "天皇誕生日"
 
         # Vernal Equinox Day
-        self[self._vernal_equinox_day(year)] = "春分の日"
+        epoch = Sun.get_equinox_solstice(year, target="spring")
+        equinox = Epoch(epoch).get_full_date()
```

Comment:
I recommend using "Asia/Tokyo" tz adjustment here mostly for consistency reasons (see Chile example) as DST is not observed in Japan.

Reply:
Updated as suggested ;) (much cleaner now ... Thanks)